extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -775,11 +775,14 @@ export class SqlDatabaseTree {
 			this._model._assessmentResults.databaseAssessments.sort((db1, db2) => {
 				return db2.issues.length - db1.issues.length;
 			});
+			// Reset the dbName list so that it is in sync with the table
+			this._dbNames = this._model._assessmentResults.databaseAssessments.map(da => da.name);
```

Comment:
If this is just a map of the names from the model seems like you should just have a property either here or in the model that gets this value - otherwise it's too easy to have the values be out of sync like they are here.

Reply:
there is a different scenario: target == sqlvm also involved. I would leave it to @aasimkhan30 and @cssuh to fix it properly as you suggested